pkg/github/pull_request_reviews.go

```diff
@@ -0,0 +1,244 @@
+//			             name
+//			             company
+//			             email
+//			             url
+//			           }
+//			           comments(first: 0) {
+//			             totalCount
+//		               }
+//				     }
+//				  }
+//				}
+//			  }
+//			}
+type QueryListPullRequestReviews struct {
+	Search struct {
+		Nodes []struct {
+			PullRequest struct {
+				Number     int64
+				Title      string
+				URL        string
+				State      githubv4.PullRequestState
+				Author     Author
+				Repository Repository
+				Reviews    struct {
+					Nodes []struct {
+						Review struct {
+							CreatedAt githubv4.DateTime
+							UpdatedAt githubv4.DateTime
+							URL       string
+							Author    Author
+							State     githubv4.PullRequestReviewState
+							Comments  struct {
+								TotalCount int64
+							} `graphql:"comments(first: 0)"`
+						} `graphql:"... on PullRequestReview"`
+					}
+					PageInfo models.PageInfo
+				} `graphql:"reviews(first: 100, after: $reviewCursor)"`
+			} `graphql:"... on PullRequest"`
+		}
+		PageInfo models.PageInfo
+	} `graphql:"search(query: $query, type: ISSUE, first: 100, after: $prCursor)"`
+}
+
+type Author struct {
+	User models.User `graphql:"... on User"`
+}
+
+type Review struct {
+	CreatedAt     githubv4.DateTime
+	UpdatedAt     githubv4.DateTime
+	URL           string
+	Author        Author
+	State         githubv4.PullRequestReviewState
+	CommentsCount int64
+}
+
+type PullRequestWithReviews struct {
+	Number     int64
+	Title      string
+	State      githubv4.PullRequestState
+	URL        string
+	Author     Author
+	Repository Repository
+	Reviews    []Review
+}
+
+// PullRequestReviews is a list of GitHub Pull Request Reviews
+type PullRequestReviews []PullRequestWithReviews
+
+// Frames coverts the list of Pull Request Reviews to a Grafana DataFrame
+func (prs PullRequestReviews) Frames() data.Frames {
+	frame := data.NewFrame(
+		"pull_request_reviews",
+		data.NewField("pull_request_number", nil, []int64{}),
+		data.NewField("pull_request_title", nil, []string{}),
+		data.NewField("pull_request_state", nil, []string{}),
+		data.NewField("pull_request_url", nil, []string{}),
+		data.NewField("pull_request_author_name", nil, []string{}),
+		data.NewField("pull_request_author_login", nil, []string{}),
+		data.NewField("pull_request_author_email", nil, []string{}),
+		data.NewField("pull_request_author_company", nil, []string{}),
+		data.NewField("repository", nil, []string{}),
+		data.NewField("review_author_name", nil, []string{}),
+		data.NewField("review_author_login", nil, []string{}),
+		data.NewField("review_author_email", nil, []string{}),
+		data.NewField("review_author_company", nil, []string{}),
+		data.NewField("review_url", nil, []string{}),
+		data.NewField("review_state", nil, []string{}),
+		data.NewField("review_comment_count", nil, []int64{}),
+		data.NewField("review_updated_at", nil, []time.Time{}),
+		data.NewField("review_created_at", nil, []time.Time{}),
```

Comment:
Not sure if I love the prefixes here, but to be the most straightforward to users this felt right?